main.go

```diff
@@ -139,6 +148,20 @@ func recordingFinalName(filename string) string {
 	return reTempName.ReplaceAllString(filename, `$1`)
 }
 
+func cameraPowerOffOn(pin string) error {
+	log.Println("Turning camera off then on.")
+	powerPin := gpioreg.ByName(pin)
+	if err := powerPin.Out(gpio.Low); err != nil {
+		return fmt.Errorf("failed to set camera power pin low: %v", err)
+	}
+	time.Sleep(2000 * time.Millisecond)
+	if err := powerPin.Out(gpio.High); err != nil {
+		return fmt.Errorf("failed to set camera power pin high: %v", err)
+	}
+	time.Sleep(6000 * time.Millisecond)
+	return nil
+}
+
 func powerupCamera(pin string) error {
```

Comment:
I wonder if we should run `cameraPowerOffOn` whenever the program starts (and get rid of `powerupCamera`). Seems like it might be good to always start from a known state.

The delay will be annoying though.